python/lsst/ts/MTMount/limits.py

```diff
@@ -62,12 +62,12 @@ def scaled(self, factor=1.01):
 
```

Comment:
I should have said this in the ticket but please, mention where these number came from.

For Elevation and Azimuth LTS-103.

For CCW LTS-218.

Reply:
Thank you. I will add that information.